pkg/digest/digest.go

```diff
@@ -34,7 +35,9 @@ func Sha256(value string) string {
 func Sha1(contents []string) string {
 	h := sha1.New()
 	for _, content := range contents {
-		io.WriteString(h, content)
+		if _, err := io.WriteString(h, content); err != nil {
+			fmt.Println(err)
```

Comment:
`fmt.Println(err)` would send error message to the stdout, I am wondering if we should send it to stderr.

In addition, I wish to see the principle that where indeed we should handle the returned error?

Reply:
I think the error in this function that should not return.
becouse this func be called in append() func.
can only output.

Reply:
I do not mean `return err`, but stream error message into stderr rather than stdout.

Reply:
I see,I fixed it.